content/blog/2022/10/2022-10-10-plugin-health-scoring-system-report.adoc

```diff
@@ -0,0 +1,133 @@
+* Jenkins Online Meetup Phase 1: Demo and Presentation
+
+video::loLSNdCv6K4[youtube,width=800,height=420,start=1089]
+
+---
+
+== Phase 2
+
+In this phase, the probe engine for the probes was designed.
+Processes like reading the update center and executing the probe engine are now running on a schedule, without the need to restart the application.
+Probes created at this time are getting run by the probe engine, and are listed on the UI to highlight the current state of the project.
+
+=== Deliverables
+
+* Design the Probe engine for the probes (link:https://github.com/jenkins-infra/plugin-health-scoring/pull/19[PR])
+* Schedule the reading of the update center and probe engine execution (link:https://github.com/jenkins-infra/plugin-health-scoring/pull/20[PR#1] and link:https://github.com/jenkins-infra/plugin-health-scoring/pull/30[PR#2])
+* List the available probes of the application (link:https://github.com/jenkins-infra/plugin-health-scoring/pull/27[PR])
+* Add a probe (link:https://github.com/jenkins-infra/plugin-health-scoring/pull/33[PR])
+* Visit link:https://plugin-health.jenkins.io[plugin-health.jenkins.io] to view a list of active probes. A big thanks to the Jenkins Infrastructure team (especially link:https://www.jenkins.io/blog/authors/hlemeur[Hervé Le Meur] and link:https://www.jenkins.io/blog/authors/dduportal[Damien Duportal]) for their help and support throughout.
+
+=== Resources
+
+* link:https://github.com/jenkins-infra/plugin-health-scoring/milestone/2?closed=1[View Milestone 2]
+* Jenkins Online Meetup Phase 2: link:https://docs.google.com/presentation/d/1HOHRVFOfH07TnBfbGh3xAqakA3NfmKni_7FYyCx-llw/edit#slide=id.p[Slides]
+* Jenkins Online Meetup Phase 2: Demo and Presentation
+
+video::fM2SMbppRxw[youtube,width=800,height=420,start=342]
+
+* A BIG shoutout to my mentor link:https://www.jenkins.io/blog/authors/jleon/[Jake Leon], for devoting his time to coaching me so well for both of these presentations.
+
+---
+
+== Next Steps
+
+- Add more probes to the project.
+- Generate the plugin health scores based on the data extracted by the probes.
+- Deploy the health scores via a JSON file, similar to how Jenkins Update Center does it.
+- Render the detailed report of the health score of each plugin by fetching the JSON data generated above.
+- [Stretch Goal] Display Plugin health score on Plugin Manager.
+
+---
+
+== Acknowledgments
+
+* I'm extremely grateful to have been given this opportunity to contribute to the Jenkins project. I owe it to my mentors for being able to help take this project forward and learn lots of things along the way. Shoutout to link:https://www.jenkins.io/blog/authors/alecharp[Adrien] and Jake. They invested tremendous time and energy in mentoring me and driving this project forward. They synced with me weekly and made sure that I was learning and that we were taking this project forward, 1 PR at a time.
```

Comment:
```suggestion
* I'm extremely grateful to have been given this opportunity to contribute to the Jenkins project. I owe it to my mentors for being able to help take this project forward and learn lots of things along the way. Shoutout to link:https://www.jenkins.io/blog/authors/alecharp[Adrien Lecharpentier] and Jake Leon. They invested tremendous time and energy in mentoring me and driving this project forward. They synced with me weekly and made sure that I was learning and that we were taking this project forward, 1 PR at a time.
```
First occurence for Adrien, nice to get his full name at least once in this article. Added "Leon" to be similar in this phrase.